eval/shell/job.go

```diff
@@ -336,6 +336,8 @@ func (j *Job) setupSimpleCmd(cmd *expr.ShellSimpleCmd, sio stdio) (*proc, error)
 	case "jobs":
 		bgList(j.Stderr)
 		return nil, nil
+	case "export":
+		return nil, export(argv[1:])
```

Comment:
`export` can also be called with no arguments.

No need to implement that now, but this shouldn't panic when len(argv) == 1.